agb/examples/output.rs

```diff
@@ -2,8 +2,9 @@
 #![no_main]
```

Comment:
I would like to avoid this if possible

Reply:
I'm not quite sure how... Adding #![no_main] to the macro output gives 
```
error: an inner attribute is not permitted in this context
  --> examples/bitmap3.rs:13:1
   |
13 | #[agb::entry]
   | ^^^^^^^^^^^^^
   |
   = note: inner attributes, like `#![no_std]`, annotate the item enclosing them, and are usually found at the beginning of source files. Outer attributes, like `#[test]`, annotate the item following them.
   = note: this error originates in the attribute macro `agb::entry` (in Nightly builds, run with -Z macro-backtrace for more info)
```

and not including it at all gives 

```
error[E0601]: `main` function not found in crate `beep`
  --> examples/beep.rs:1:1
   |
1  | / #![no_std]
2  | |
3  | | extern crate agb;
4  | |
...  |
37 | |     loop {}
38 | | }
   | |_^ consider adding a `main` function to `examples/beep.rs`
```

Reply:
It's fine actually, I've had a look at cortex crate examples ([example](https://github.com/rust-embedded/cortex-m-rt/blob/master/examples/data_overflow.rs)) which have #[entry] and #![no_main]